tests/snappi_tests/utils/examples.py

```diff
@@ -0,0 +1,69 @@
+import logging
```

Comment:
I just realized that this will be helpful to other regular test cases too, so maybe we should move it to the tests/common/telemetry folder.

Reply:
Do you mean moving the file (examples.py) to tests/common/telemetry/? There is no folder named telemetry under tests/common/ as of now. Do you want me to create one now? Or do this moving after tests/common/Telemetry is created? This file does not include any user interfaces. So we can postpone the moving.

Reply:
yes, let's create one.

Reply:
this is duplicated now, we should remove it.

Reply:
and by moving the file, I mean move the metrics.py and report_factory.py, not just the example : D